metagraph/src/graph/alignment/dbg_aligner.hpp

```diff
@@ -23,17 +23,26 @@ class IDBGAligner {
     typedef Alignment<node_index> DBGAlignment;
     typedef QueryAlignment<node_index> DBGQueryAlignment;
     typedef typename DBGAlignment::score_t score_t;
+    typedef std::function<void(std::string_view /* header */,
+                               std::string_view /* seq */,
+                               bool /* orientation */)> QueryCallback;
```

Comment:
ha, perfect, you already have this guy and don't have to redefine (see my comment above).